readthedocs/settings/base.py

```diff
@@ -264,6 +264,11 @@ def USE_PROMOS(self):  # noqa
             'schedule': crontab(minute=0, hour='*/3'),
             'options': {'queue': 'web'},
         },
+        'every-week-endpoint-deprecated-notifications': {
+            'task': 'readthedocs.projects.tasks.send_deprecated_endpoint_notifications',
+            'schedule': crontab(minute=0, hour='10', day_of_week='monday'),
+            'options': {'queue': 'web'},
+        },
```

Comment:
I don't think we need a task here, messages will just go out from our notification system.